#ifndef CV_TEST_NODE_H
#define CV_TEST_NODE_H

#include <ros/ros.h>
#include <ros/callback_queue.h>
#include <image_transport/image_transport.h>
#include <cv_bridge/cv_bridge.h>
#include <sensor_msgs/image_encodings.h>
#include <list>

class CVTestNode {

	private:
	
	public:
	
	private:
	
};
	
#endif
